CVE-2017-12617/cvex.yml

```diff
@@ -0,0 +1,14 @@
+blueprint: ubuntu2204
```

Comment:
Use the ubuntu2204-ubuntu2204 blueprint. Install and run the Tomcat on ubuntu1, copy the exploit to ubuntu2, run the exploit on ubuntu2 that would attack ubuntu1. Make sure that router_raw.pcap captures the network traffic. The goal is to get the traffic! Take a look at a good example that is also targeting Apache Tomcat: https://github.com/ucsb-seclab/CVEX-records/tree/main/CVE-2020-1938